src/Templates/MLDotNet/Microsoft.VisualStudio.Templates.MLDotNet/Content/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,87 @@
+                    mlContext,
+                    new TextLoader.Arguments()
+                    {
+                        Separator = "tab",
+                        HasHeader = true,
+                        Column = new[]
+                        {
+                            new TextLoader.Column("Label", DataKind.Bool, 0),
+                            new TextLoader.Column("Text", DataKind.Text, 1)
+                        }
+                    });
+
+            IDataView trainingDataView = reader.Read(new MultiFileSource(@"Data\wikipedia-detox-250-line-data.tsv"));
+            // 2. Create a pipeline to prepare your data, pick your features and apply a machine learning algorithm.
+            // 2a. Featurize the text into a numeric vector that can be used by the machine learning algorithm.
+            var pipeline = mlContext.Transforms.Text.FeaturizeText("Text", "Features")
+                    .Append(mlContext.BinaryClassification.Trainers.StochasticDualCoordinateAscent( "Label", "Features"));
+
+            // 3. Get a model by training the pipeline that was built.
+            var model = pipeline.Fit(trainingDataView);
+
+            // 4. Evaluate the model to see how well it performs on different data (output the percent of examples classified correctly).
+            Console.WriteLine("Training of model is complete \nTesting the model with test data");
+            IDataView testDataView = reader.Read(new MultiFileSource(@"Data\wikipedia-detox-250-line-test.tsv"));
+            var predictions = model.Transform(testDataView);
+            var binClassificationCtx = new BinaryClassificationContext(mlContext);
+            var metrics = binClassificationCtx.Evaluate(predictions, "Label");
+            Console.WriteLine($"Accuracy: {metrics.Accuracy:P2}");
+
+            // 5. Use the model for a single prediction.
+            var predictionFunct = model.MakePredictionFunction<SentimentData, SentimentPrediction>(mlContext);
+            var testInput = new SentimentData { Text = "ML.NET is fun, more samples at https://github.com/dotnet/machinelearning-samples" };
+            SentimentPrediction resultprediction = predictionFunct.Predict(testInput);
+
+            /* This template uses a minimal dataset to build a sentiment analysis model which leads to relatively low accuracy. 
+             * Building good Machine Learning models require large volumes of data. This template comes with a minimal dataset (Data/wikipedia-detox) for sentiment analysis. 
+             * In order to build a sentiment analysis model with higher accuracy please follow the walkthrough at https://aka.ms/mlnetsentimentanalysis/. */
+            Console.WriteLine($"Predicted sentiment for \"{testInput.Text}\" is: { (Convert.ToBoolean(resultprediction.Prediction) ? "Positive" : "Negative")}");
+
+            // 6. Save the model to file so it can be used in another app.
+            Console.WriteLine("Saving the model");
```

Comment:
@eerhardt What about the code for saving the model on disk? We don't have this in our getting started sample on the marketing page. I think we shouldn't have it as part of getting started. Developers can discover through docs/ samples. What do you think?

Reply:
I'm not sure I have a strong opinion one way or the other. Here it is nice because it explicitly calls out other things you can do with it.

For example, in the marketing page getting started, we don't Evaluate the model either.

So I'm fine with doing a couple more operations in this template, I think.  But if you feel strongly the other way, I wouldn't object.

Reply:
I thought about it more. Let's keep it. In the marketing page we will won't show Evaluate and model saving.